lib/jsonToCsv.js

```diff
@@ -2,11 +2,22 @@ var through = require('through2');
 
 module.exports = function(){
   var stream = through(function(chunk, enc, cb){
-    var obj = JSON.parse(chunk.toString());
-    var coords = obj.geometry.coordinates;
-    var props = obj.properties;
 
-    return cb(null, [coords[0], coords[1], props.address, props.alt_address].join(',') + '\n');
+    try{
+      var obj = JSON.parse(chunk.toString());
+      var coords = obj.geometry.coordinates;
+      var props = obj.properties;
+      var x = coords[0];
+      var y = coords[1];
+      var addr = props.address;
+      var alt = props.alt_address;
```

Comment:
Why are we still including `alt_address`?  Haven't we abandoned that?  Have we ever used it for anything?